app/controllers/service_controller.rb

```diff
@@ -252,42 +158,14 @@ def textual_tower_job_group_list
   end
   helper_method :textual_tower_job_group_list
 
-  def features
-    [
-      {
-        :role     => "service",
-        :role_any => true,
-        :name     => :svcs,
-        :title    => _("Services")
-      }
-    ].map { |hsh| ApplicationController::Feature.new_with_hash(hsh) }
-  end
-
-  def service_ownership
-    assert_privileges('service_ownership')
-    @explorer = true
-    set_ownership
-    replace_right_cell(:action => 'ownership')
-  end
-
-  def service_tag_edit
-    assert_privileges('service_tag')
-    @explorer = true
-    service_tag
-    replace_right_cell(:action => 'tag')
-  end
-
-  def service_retire
-    assert_privileges('service_retire')
-    @explorer = true
-    retirevms
-    replace_right_cell(:action => 'retire')
-  end
+  # def service_retire
+  #   assert_privileges('service_retire')
+  #   retirevms
+  # end
 
-  def service_retire_now
-    @explorer = true
-    retirevms_now
-  end
+  # def service_retire_now
```

Comment:
@jeffibm You have some commented out lines in this file, should these be removed?